src/ui/3d/qgspointcloud3dsymbolwidget.ui

```diff
@@ -75,6 +75,13 @@
         </property>
        </widget>
       </item>
+      <item row="2" column="0" colspan="2">
+       <widget class="QCheckBox" name="mShowBoundingBoxesCheckBox">
+        <property name="text">
+         <string>Show point cloud bounding boxes</string>
```

Comment:
I would rephrase this to just "Show bounding boxes" - it should be clear it is for point clouds